app/scss/rewards.scss

```diff
@@ -25,6 +25,7 @@
 
 #ghostery-shadow-root {
 	opacity: 0;
+	transition: 'opacity 200ms ease-in';
```

Comment:
guys, @christophertino @trickpattyFH20 
I could not find who sets `resolved` class, so solved issue with `transition and setTimeout` see above please